multi_asset/multi_asset.md

```diff
@@ -266,3 +274,32 @@ implement the `multi_token_receiver`.
 
 Only non-standard transfer functions MAY allow tokens to be sent to a recipient
 contract that does NOT implement the necessary `multi_token_receiver` hook functions.
+
+#### Token owner contract implementation guidelines
+
+This specification focuses on token transfer logic only. Implementation of the
+actual token receiver contract may differ depending on the particular business
+use-case. However, potential locking of tokens on the receiver account must be
+taken into consideration.
+
+Token owner contract MUST implement `multi_token_receiver` interface to receive
+tokens.
+
+By default, transfer **from** a token owner to another contract can be
+initiated only by the owner itself. If owner contract implementation does
+not allow to initiate such transfer and/or add operators (which also can be
+performed only by the owner), tokens will remain locked on such contract forever.
+
+There are a few possible ways to enable transferring tokens from a token owner
+contract. The concrete implementation of multi asset contract and/or owner
+contract may use the following strategies or their combination:
+
+  1. Add administrative entry points to multi asset contract to allow burn
+  tokens for any token owner.
```

Comment:
Can you elaborate, what would be the workflow to free locked tokens using such an entry point?

Reply:
Burn locked tokens away...

Reply:
then the locked tokens would actually be gone - probably not the desired outcome

Reply:
On a side note, token owner contract that acts as a black hole and locks tokens on purpose may actually be a useful construct in some applications :)

Reply:
This strategy is a building block. I should not invent plausible scenario where it can be used, but...
If the owner wants to redeem his tokens for real goods, administrators might burn tokens